scripts/effects/dynamic-effects-adder.js

```diff
@@ -62,6 +68,42 @@ export default class DynamicEffectsAdder {
     });
   }
 
+  _addEnlargeReduceEffect(effect, actor) {
```

Comment:
I think I'd prefer to have these handled separately with some common private methods as needed